website/pages/api-docs/acl/auth-methods.mdx

```diff
@@ -40,13 +40,18 @@ The table below shows this endpoint's support for
 - `Name` `(string: <required>)` - Specifies a name for the ACL auth method. The
   name can contain alphanumeric characters, dashes `-`, and underscores `_`.
   This field is immutable and must be unique.
+
 - `Type` `(string: <required>)` - The type of auth method being configured.
   The only allowed value in Consul 1.5.0 is `"kubernetes"`. This field is
   immutable.
 
 - `Description` `(string: "")` - Free form human readable description of the
   auth method.
 
+- `DisplayName` `(string: "")` - An optional name to use instead of the `Name`
+  field when displaying information about this auth method in any kind of user
```

Comment:
`in any kind of user interface` isn't this primarily used in the UI and not CLI and we should specify that as much? 🤔

Reply:
Well you could roll your own wrapper on top of consul if you didn't want to use our UI.

I could just end the sentence at "...about this auth method." instead to keep it vague.

Reply:
o ok sgtm